pkg/aggregator/aggregator.go

```diff
@@ -228,16 +238,22 @@ func MergeSpecs(dest, source *spec.Swagger) error {
 }
 
 func mergeSpecs(dest, source *spec.Swagger, renameModelConflicts, ignorePathConflicts bool) (err error) {
-
 	specCloned := false
 	if ignorePathConflicts {
 		keepPaths := []string{}
+		hasConflictingPath := false
 		for k := range source.Paths.Paths {
 			if _, found := dest.Paths.Paths[k]; !found {
 				keepPaths = append(keepPaths, k)
+			} else {
+				hasConflictingPath = true
```

Comment:
isn't `hasConflictingPath` and `len(keepPaths) != 0` equal? Don't understand why we need these two way to express that.

Reply:
If there is no common paths, `len(keepPaths) != 0` but `hasConflictingPath` will be false.In the case of no conflicting path, we want to avoid an unnecessary Clone.

Reply:
misread. It's logically different. thanks.